content/2017.04.getting-started-with-elm.rst

```diff
@@ -0,0 +1,433 @@
+Getting started with Elm
+########################
+
+:lang: en
+:date: 2017-04-13
+:url: getting-started-with-elm
+:summary: What is Elm? how does it work? How can I try it really fast?
+
+Introduction
+============
+
+The past two weeks we have been traveling and meeting a bunch of
+developers.
+
+It was a chance for us to tell them about elm through talks at
+conference and to train a little and short one hour workshop.
+
+You can find the slides of the two talks here:
+
+- `a short introduction to Elm by Niko <http://slides.com/n1k0/elm>`_
+- `Talking JSON with Elm by Natim <http://natim.github.io/django-slides/elm-json/>`_
+
+However, this article is a summary of the take aways of this workshop
+for thoose who would like to know a bit more about Elm.
+
+
+What is Elm and why should I be concerned?
+==========================================
+
+Elm is a toolkit that is focused on making sure your app state and the
+DOM (html nodes) that reflects it are always in sync and that updates
+on your app state are reflected on what the user is seeing.
```

Comment:
Capitalize `HTML`. I feel like the parenthesized remark before `that reflects it` make it a little hard to keep track of what's going on. Why not just `your app state and the HTML that reflects it`? Also, I feel like the second part of this paragraph (`... updates on your app state are reflected on what the user is seeing`) is a bit redundant with the first part.